src/lorawan_utils.erl

```diff
@@ -159,7 +159,6 @@ event_args(Event) when is_atom(Event) ->
 event_args(Event) when is_binary(Event) ->
     {Event, undefined}.
 
-inc(undefined) -> 1;
```

Comment:
This has a reason. The dialyzer is wrong when saying this will never be used.

Reply:
OK, I'm removing this commit.